project/accounts/api.py

```diff
@@ -18,12 +18,13 @@
 from accounts.permissions import IsProfileOwnerOrDuringRegistrationOrReadOnly
 from accounts.serializers import ProfileSerializer, ProfileListSerializer
 from accounts.forms import UpdateProfileImage
-from api.utils import get_account
-from api.models import Thread, Civi, Activity
+from accounts.utils import get_account, json_response

```

Comment:
```suggestion
from threads.utils import json_response
from accounts.utils import get_account
```

`json_response` is in the `threads.utils` file not in the `accounts.utils`.